api/controllers/pinsV2.js

```diff
@@ -0,0 +1,301 @@
+// Imports
+const defaultLog = require('winston').loggers.get('default');
+const Actions    = require('../helpers/actions');
+const projectDAO = require('../dao/projectDAO');
+const pinDAO     = require('../dao/pinDAO');
+
+// Constants
+const PUBLIC_ROLES = ['public'];
+const SECURE_ROLES = ['sysadmin', 'staff'];
+// Vars
+/* put any needed local variables here */
+// functions
+/* put any needed local functions here */
+// Exports
+
+// OPTIONS
+exports.pinOptions = function (args, res, rest) 
```

Comment:
What is "rest" on these two functions? Isn't this the next function passed in from express?

Reply:
It should be "next". Looks like a copy/paste error from the original project controller code (it's in all of the old controllers actually). Will be updated to "next"